### ISSUE TITLE
warning: cast from ‘unsigned char*’ to ‘uint16_t*’ {aka ‘short unsigned int*’} increases required alignment of target type [-Wcast-align]

### ISSUE THREAD

**Issue by malaterre** [team-charls/charls#160]
Posting as issue for now:

```
/<<PKGBUILDDIR>>/test/performance.cpp: In function ‘void {anonymous}::test_file16_bit_as12(const char*, int, rect_size, int, bool)’:
/<<PKGBUILDDIR>>/test/performance.cpp:34:19: warning: cast from ‘unsigned char*’ to ‘uint16_t*’ {aka ‘short unsigned int*’} increases required alignment of target type [-Wcast-align]
   34 |     auto* const p{reinterpret_cast<uint16_t*>(uncompressed_data.data())};
      |                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[100%] Linking CXX executable charlstest
```

**Comment by malaterre** [comment]
* https://buildd.debian.org/status/fetch.php?pkg=charls&arch=mips64el&ver=2.3.1%2Bdfsg-1&stamp=1643190949&raw=0

**Comment by vbaderks** [comment]
This has been resolved with commit 2991763583b1dc065da71d096c4dc9b806646365
